main.go

```diff
@@ -44,6 +46,11 @@ func main() {
 	handler, stop := web.NewHandler(config.Web, remote, consulInstance)
 	defer stop()
 
+	if config.SSE.Enable {
+		stopSSE := sse.NewHandler(config.Web, remote, consulInstance)
```

Comment:
I wonder if enabling SSE should disable at the same time legacy event bus handler, so there will be no chance for race between event bus and SSE... what do you guys think, @tomez @janisz?